window/settings_panel.py

```diff
@@ -440,7 +440,7 @@ def buildDatabaseBtnContextMenuEvent(self, point: QPoint) -> None:
         )  # db builder expiry
         set_steam_apikey = contextMenu.addAction(
             "Set Steam WebAPI key"
-        )  # steam webapi key
+        )  # steam web api key
```

Comment:
Technically, this should be "API-key" stuck_out_tongue

Reply:
done